### ISSUE TITLE
failed with refusing to fetch...

### ISSUE THREAD

**Issue by bruffridge** [repo-sync/pull-request#10]
```yaml
    - uses: actions/checkout@v1
      with:
        ref: develop
    - name: pull-request
      uses: repo-sync/pull-request@v2
      with:
        source_branch: develop
        destination_branch: master
        pr_label: automerge
        github_token: ${{ secrets.GITHUB_TOKEN }}
```

fails with:

```
##[error]Docker run failed with exit code 128
Run repo-sync/pull-request@v2
/usr/bin/docker run --name e4e3a6913e7162e7d458b897f452bfbccfa89_b5ac7c --label 2e4e3a --workdir /github/workspace --rm -e INPUT_SOURCE_BRANCH -e INPUT_DESTINATION_BRANCH -e INPUT_PR_LABEL -e INPUT_GITHUB_TOKEN -e INPUT_PR_TITLE -e INPUT_PR_BODY -e INPUT_PR_REVIEWER -e INPUT_PR_ASSIGNEE -e INPUT_PR_MILESTONE -e GITHUB_TOKEN -e HOME -e GITHUB_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_ACTOR -e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF -e GITHUB_EVENT_NAME -e GITHUB_WORKSPACE -e GITHUB_ACTION -e GITHUB_EVENT_PATH -e RUNNER_OS -e RUNNER_TOOL_CACHE -e RUNNER_TEMP -e RUNNER_WORKSPACE -e ACTIONS_RUNTIME_URL -e ACTIONS_RUNTIME_TOKEN -e GITHUB_ACTIONS=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/nasawds-2.0/nasawds-2.0":"/github/workspace" 2e4e3a:6913e7162e7d458b897f452bfbccfa89
fatal: Refusing to fetch into current branch refs/heads/develop of non-bare repository
##[error]Docker run failed with exit code 128
```

Any ideas?

**Comment by bruffridge** [comment]
had to checkout master branch instead of develop, and then it worked.

**Comment by ChristophShyper** [comment]
Getting the same error using `actions/checkout@v2`.
Works only with `actions/checkout@v1`.

**Comment by wei** [comment]
Fixed in #24
